containers/wordpress/README.md

```diff
@@ -30,7 +30,8 @@ a baseline. The following modifications were made to wp_base in addition to cont
   * MariaDB now uses large pages and additional tuning
   * NUMA optimization/multi instance (must be done via pinning, see below)
     * Note that for NUMA optimization/pinning you may do this with the base container if you wish to isolate this optimization.
-* wp5.2_base_http builds wp4.2_php7.4_base_querycacheoff_http, but uses WordPress 5.2 and its associated database dump and URLs.  
+* wp5.2_base_http builds upon wp4.2_php7.4_base_querycacheoff_http, but uses WordPress 5.2 and its associated database dump and URLs.  
```

Comment:
Could we add more detail to the names to make them consistent with the 4.2/7.4 containers?

Reply:
I suppose this is a comment specific to the bullets in the readme